package.json

```diff
@@ -10,6 +10,8 @@
         "build": "lerna run build",
         "clean": "lerna run clean",
         "deploy": "NODE_DEBUG=gh-pages gh-pages -d packages/docs/dist -b master",
+        "format": "prettier --write \"./**/*.{js,json,md,ts,tsx,yml}\"",
+        "format-check": "prettier --check \"./**/*.{js,json,md,ts,tsx,yml}\"",
```

Comment:
this is not used in CI yet, but I imagine it being used the same as Blueprint's repo after we migrate to ESLint in a future PR.